app/package.json

```diff
@@ -28,6 +26,6 @@
     "serve": "parcel serve index.html --out-dir ../dist/ --no-cache",
     "watch": "npm run watch:script",
     "sync-assets": "copy-aragon-ui-assets ../dist && copyfiles -u 1 './public/**/*' ../dist",
-    "start": "npm run sync-assets && npm run watch:script & npm run devserver"
+    "start": "npm run sync-assets && npm run watch:script & npm run serve"
```

Comment:
Not sure, but I think this is correct (this package.json doesn't contain a `devserver` script).

Reply:
Good catch.